src/renderer/components/chat/ChatListHelpers.js

```diff
@@ -73,6 +73,8 @@ export const useLazyChatListItems = chatListIds => {
 
   const chatIdsInView = (offset) => {
     let [indexStart, indexEnd] = getIndexStartEndInView()
+    // load 10 more than in view to have a buffer when scrolling
+    indexEnd += 10
```

Comment:
The offset option is actually for exactly this case. So you can just increase offset when the chatIdsInView method is getting called.